src/components/user/UserProfile.js

```diff
@@ -79,6 +114,54 @@ export default function PersonalProfile() {
           View Recommendations
         </Button>
       </Link>
+
+      <h3 className="text-uppercase">Purchased Albums</h3>
+      {error && <Alert severity="error">{error}</Alert>}
+
+      {loading && <p>Loading purchased albums...</p>}
+      <Table className="text-center mt-5">
+        <tbody>
+          <TableRow>
+            <TableCell>
+              {' '}
+              <b>
+                {' '}
+                <em></em>
+              </b>
+            </TableCell>
+            <TableCell>
+              <b>
+                {' '}
+                <em>Artist Name</em>
+              </b>
+            </TableCell>
+            <TableCell>
+              <b>
+                {' '}
+                <em>Album Name</em>
+              </b>
+            </TableCell>
+            {/* Add more cells for other field names */}
+          </TableRow>
+          {purchasedAlbums.map((purchasedAlbum) => (
+            <TableRow key={purchasedAlbum._id} style={{ padding: '0' }}>
+              <div>
+                <TableCell>
+                  <img
+                    src={purchasedAlbum.album.image}
+                    alt="albumImage"
+                    height="100px"
+                    width="100px"
+                  />
+                </TableCell>
+              </div>
```

Comment:
I don't think we need this extra `div` wrapping the table cell -- it also kind of breaks the border line between the rows right now:

```suggestion
              <TableCell>
                <img
                  src={purchasedAlbum.album.image}
                  alt="albumImage"
                  height="100px"
                  width="100px"
                />
              </TableCell>
```